File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ""
    j = 0
    for i, c in enumerate(s):
        if c == " ":
            j = -1
        else:
            if j % 2 == 0:
                c = c.upper()
            else:
                c = c.lower()
                
        j += 1
        answer += c
    return answer

File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.cs
public class Solution {
    public string solution(string s) {
        string answer = "";
        int j = 0;
        
        for (int i = 0; i < s.Length; i++)
        {
            string now = s[i].ToString();
            
            if (now.StartsWith(' '))
                j = -1;
            else
                if (j % 2 == 0)
                    now = now.ToUpper();
                else
                    now = now.ToLower();
            
            answer += now;
            j += 1;
        }
        
        return answer;
    }
}